src/isomorphic/classic/class/__tests__/ReactClassMixin-test.js

```diff
@@ -263,7 +261,7 @@ describe('ReactClass-mixin', function() {
     expect(Component.abc).toBe('def');
   });
 
-  it("should throw if mixins override each others' statics", function() {
+  it('should throw if mixins override each others\' statics', function() {
```

Comment:
Let's do avoid-escape http://eslint.org/docs/rules/quotes.html and then leave things like these as double quotes.